src/serialise/ahoXml/generate.ts

```diff
@@ -533,26 +479,39 @@ const mapAhoCXE01ToXml = (pncQuery: PncQueryResult): Cxe01 => ({
     : undefined
 })
 
-const mapAhoToXml = (aho: AnnotatedHearingOutcome): AhoXml => {
+const mapAhoToXml = (aho: AnnotatedHearingOutcome, validate = true): AhoXml => {
```

Comment:
I don't think the typing is correct for this function now - if validate is false, it will return a `Br7AnnotatedHearingOutcome` I think

Reply:
It does seem to fit the pre-existing type:

```
export interface AhoXml {
  "?xml": XML
  "br7:AnnotatedHearingOutcome"?: Br7AnnotatedHearingOutcome
  "br7:HearingOutcome"?: Br7HearingOutcome
}
```